### ISSUE TITLE
fine-tune settings

### ISSUE THREAD

**Issue by hengrui0516** [moured/YOLOv10-Document-Layout-Analysis#3]
Thanks for your work!
I wonder your finetune settings on doclaynet.
And I find that "The models were fine-tuned using 4xA100 GPUs on the Doclaynet-base dataset, which consists of 6910 training images, 648 validation images, and 499 test images." in readme.md. As far as I know, Doclaynet dataset is ten times bigger than those, containing about 69103 training images, 6480 validation images, and 4994 test images.

**Comment by moured** [comment]
Hello @hengrui0516 , thanks for the valuable comment. You're right; I missed the last digit when writing the README. The models were definitely trained on the whole dataset. I have updated* the correct values.

**Comment by sunzx8** [comment]
> Hello @hengrui0516 , thanks for the valuable comment. You're right; I missed the last digit when writing the README. The models were definitely trained on the whole dataset. I will upload the correct values right away.

hi，could you tell me the hypermeters during training like epoch, imgsize,batch size? thank you!

**Comment by moured** [comment]
sure @sunzx8,

epochs: 100
batch: 64
imgsz: 640

**Comment by aalhayali** [comment]
Hi! What were the labels used for this model? Is there a repo where I can play around the code further than inference?

EDIT: I assume it's based on the labels provided in the dataset you pre-trained YOLO on. The labels are:

![WhatsApp Image 2024-09-19 at 18 35 22_8d60025a](https://github.com/user-attachments/assets/10e03e67-a411-44f9-a772-38c8e1e82f4e)

**Comment by moured** [comment]
@aalhayali exactly, I used the 11 classes which are provided with the DocLayNet dataset. If you wanna further modify and edit the model/training setting please refer to the Yolo-v10 repo : https://github.com/THU-MIG/yolov10
